sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -397,8 +397,7 @@ public async Task ExtractKeyPhrasesTest()
             TextAnalyticsClient client = GetClient();
             string document = "My cat might need to see a veterinarian.";
 
-            Response<IReadOnlyCollection<string>> response = await client.ExtractKeyPhrasesAsync(document);
-            IReadOnlyCollection<string> keyPhrases = response.Value;
+            KeyPhraseCollection keyPhrases = await client.ExtractKeyPhrasesAsync(document);
```

Comment:
Not sure if we should be concerned with implicit casts here, too, as some users use tests as samples.

Reply:
Will leave this PR as is and once it is in the guidelines I will address this